charts/kueue/templates/_helpers.tpl

```diff
@@ -47,7 +47,7 @@ Selector labels
 */}}
 {{- define "kueue.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "kueue.name" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
Can you explain why this is removed?

Reply:
Mostly for consistency with the Kustomize manifests. Also the only case where I could this be needed would be with two different Kueue Deployment instances in the same namespace, which I doubt is something is supported nor we want to support?

Reply:
I see, I'm not sure we want to support it (I would prefer not too, but didn't do archeology on this). 

If the change is not strictly needed for the functionality of this PR I would suggest to drop it here, and move to a dedicated cleanup PR, maybe there are other inconsistencies which could be handled there?

Reply:
Good idea. Added back in cc5dc1b1eeb710393f23b3ebff2c97c9612fdf9f.

There are the namespace fields set for cluster-scoped resources, like MutatingWebhookConfiguration, that could be cleaned up as well.